modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,34 @@
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+Running as a user namespace means that your container runs as root (user id 0) but as a user on the host. OpenShift uses CRI-O as its container engine. CRI-O features a workload feature to be able to let the pod know that it wants to run user namespaced.
```

Comment:
```suggestion
When you run <something> in a user namespace, your container runs as root (user id 0) but has user privileges on the host. {product-title} uses CRI-O as its container runtime interface. CRI-O has a workload feature to run a pod in the user namespace.
```
In this suggestion, I've applied the following sources or principles:
- Official definition of CRI-O: dehttps://cri-o.io/#:~:text=What%20is%20CRI%2DO%3F,as%20the%20runtime%20for%20kubernetes.
- Minimalism.